bitnami/kong/templates/metrics-service.yaml

```diff
@@ -18,19 +19,11 @@ metadata:
     {{- end }}
   {{- end }}
```

Comment:
Missing `metrics.service.type`

Reply:
Also missing references to other standard service elements.

Reply:
Hardcoded to use a ClusterIP service.

Reply:
The rest of parameters that make sense were added.